### ISSUE TITLE
Help! MouseIsPressed with multiple images

### ISSUE THREAD

**Issue by s-vang** [processing/p5.js#2272]
#### Nature of issue?

- [ ] Found a bug
- [ ] Existing feature enhancement
- [ x] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core
- [ ] Data
- [ ] Events
- [x ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Which platform were you using when you encountered this?

- [ ] Mobile/Tablet (touch devices)
- [ x] Desktop/Laptop
- [ ] Others (specify if possible)

#### Details about the bug: 

- p5.js version: 
- Web browser and version: 
- Operating System: 
- Steps to reproduce this:

#### New feature details:

Hello, I'm quite new to P5 and GH, so i'm not sure if this is the right place to ask, if it isn't, please direct me to the correct forum.

Basically, every time the mouse is pressed, I want a new image to pop up but keep the original image. I have 11 images that I would like to pop up after each click. Any help is greatly appreciated.

**Comment by lmccart** [comment]
the github issues here are a good place for bugs and issues with the p5.js library itself. but for questions like this one, or others about writing your own code, tests, or following tutorials, the [forum](https://forum.processing.org/two/) is the best place to post. thanks!
